test_reporting/kusto/setup.kql

```diff
@@ -183,7 +183,7 @@
 ###############################################################################
 // Create table command
 ////////////////////////////////////////////////////////////
-.create table ['RebootTimingData'] (ReportId: string, ['RebootType']:string,
+.create table ['RebootTimingData'] (ReportId: string, TrackingId: string, TestbedName: string, BaseImage:string, TargetImage:string, HwSku:string, ['RebootType']:string,
```

Comment:
Similar changes are needed to `RawRebootTimingData` table too, for the same reason as this update.

Reply:
We currently don't have `BaseImage`, `TargetImage`, and `HwSku` in the `RawRebootTimingData` table; do we want to add it there as well?

I didn't copy these fields there because there is already a shared `ReportId` column that can be used to link entries between the two tables, so it's just duplication of data.

Reply:
I agree that ReportId can be used to reference the additional info. Closing this comment.